src/pages/Edital/EnrolledList/EnrolledsList.tsx

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+import { Card, CardContent, CardHeader, Divider, Grid } from "@mui/material";
+import { useContext, useEffect, useState } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import { DataGrid, GridColDef, GridEventListener } from "@mui/x-data-grid";
+import { IADetalhes, IDetails } from "../Detalhes/Interfaces";
+import getDetailsProcessoSeletivo from "../Detalhes/Service";
+import getEnrolledList from "../Detalhes/ServiceEnrolledList";
+import UserContext from "../../../context/UserContext";
+
+export default function EnrolledsList() {
+  const navigate = useNavigate();
+  const [edital, setEdital] = useState<IDetails | undefined>();
+
+  const { editalId } = useParams();
+
+  const { user } = useContext(UserContext);
+
+  const [enrolledList, setEnrolledList] = useState<IADetalhes[]>([]);
+
+  useEffect(() => {
+    if (user)
+      getEnrolledList(editalId, user?.token).then(({ data }) =>
+        setEnrolledList(data)
+      );
+  }, [editalId, user]);
+
+  // const [loading, setLoading] = useState<boolean>(true);
+
+  const handleRowClick: GridEventListener<"rowClick"> = (params) => {
+    navigate(`/edital/${editalId}/inscritos/${params.row.id}`);
+  };
+
+  useEffect(() => {
+    getDetailsProcessoSeletivo(editalId)
+      .then(({ data }) => {
+        setEdital(data);
+      })
+      .catch(() => {
+        // TODO: Ver como exibir erros va View
+      });
+  }, [editalId]);
+
+  const colunas: GridColDef[] = [
+    { field: "id", headerName: "ID", width: 50 },
```

Comment:
Não acho que deveríamos mostrar o ID na tela

É algo que vamos usar só enquanto devs, o prof q for revisar não precisa ver